### ISSUE TITLE
'How likely are you recommend....' radio group sections is not accessible using keyboard present under SSMA Feedback window: A11y_WinForm.NETRuntime_Keyboard.

### ISSUE THREAD

**Issue by shindepra** [dotnet/winforms#13754]
"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."

### GitHub Tags:
#A11ySev2; #A11yTCS; #WCAG2.1.1; #Win32; #DesktopApp; #Win11; #WinForms .NET Runtime; #BM_WinForms.NETRuntime_Win32_June2025; #Keybaord; #A11yMAS

### Environment Details:
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061  

### Repro Steps:

1. Open SQL Server Migration Assistant for oracle.  
2. Splash screen appears and SSMA is launched  
3. Navigate to connect to oracle and connect to target button and connect it.  
4. Navigate to the 'Tools' menu and select it.  
5. Navigate to 'Convert schema' menu item and select it.  
6. Navigate to the SSMA Feedback window 
7. Verify whether 'How easy or difficult and how valuable was the code radio group sections are accessible using keyboard or not. 

### Actual Result:
'How likely are you recommend....' radio group sections is not accessible using keyboard present under SSMA Feedback window.

### Expected Result:
'How likely are you recommend....' radio group sections should be accessible using keyboard present under SSMA Feedback window.

### User Empact:
Keyboard users will be impacted if the keyboard focus doesn't land on interactive controls. Keyboard users will be blocked as they cannot access the controls. They cannot invoke and access the content and controls of it.

Sample App used:-

[RadioButtonFocusDemo.zip](https://github.com/user-attachments/files/21483118/RadioButtonFocusDemo.zip)

### Attachment:

<img width="896" height="488" alt="Image" src="https://github.com/user-attachments/assets/da1c5a5f-b718-4e09-9114-cf09d11b2989" />

https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-kkumar2_microsoft_com/EZ0H-YKcdNNOk8_8w2bp5BQBlJLS3TgOc9LeGWsuEAME7g?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0NvcHkifX0&e=lXjxlK

**Comment by LeafShi1** [comment]
RadioButton needs to set TabStop=true to focus via the Tab key

This should not be a Winform problem, this page may need to be redesigne

**Comment by i-gagan9** [comment]
Hi @LeafShi1,

The product team wants the radio button group to start with no option selected, so users are required to make a choice.

However, in WinForms, this setup runs into a limitation:
If the radio button group has no selection, it becomes non-navigable by keyboard. And if we make it focusable, it automatically selects the first radio button—which defeats the purpose.

We tried several workarounds:
Programmatically unchecking all radio buttons
Adding a hidden dummy radio button
Shifting focus to another control
Setting TabStop = true
But none of these prevent the first visible radio button from appearing selected when the group receives focus.

WinForms Limitation:
You can’t have a keyboard-navigable radio button group with no selection state.

**Comment by LeafShi1** [comment]
That's because the TabIndex of the GroupBox takes precedence over other controls on the page. You just need to adjust the order of TabIndex so that GroupBox is not in the first place.

https://github.com/user-attachments/assets/dad44c79-c748-458e-931a-e51ee94bb072

**Comment by merriemcgaw** [comment]
Also note that this product runs on .NET Framework, and this repo is about .NET 10+. Please file internal bugs in AzDO for issues related to .NET Framework.

@shindepra - I think this issue can be closed based on the investigation done by @LeafShi1.
